CONTRIBUTING.md

```diff
@@ -77,7 +84,10 @@ There are a handful of reasons the CI may have turned red. Try the following fix
 4. review `source.errors` to see if it matches our expectations
     - this isn't exactly true, but I conceptualize `source.errors` as the output if a user ran a `lint` command on `source.wdl`
     - while reviewing, ask yourself if the printed diagnostics are clear and informative
-5. repeat
+5. consider if your new rule relates to existing ones and implement the `related_rules()` method accordingly. See
+[how-should-i-decide-which-rules-to-link-using-related_rules()?](https://github.com/stjude-rust-labs/wdl/blob/main/CONTRIBUTING.md#how-should-i-decide-which-rules-to-link-using-related_rules)
```

Comment:
Can you reformat this so it's an internal link and looks a bit nicer? I'm forgetting the markdown syntax for linking to a header in the same document, but there is a way to do that.